pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -205,7 +205,12 @@ sub _suggest_kernelcare {
         }
 
         $suggestion = ($suggestion) ? '<p/><p/>' . $suggestion : '';
-        $promotion = $self->_lh->maketext('KernelCare provides an easy and effortless way to ensure that your operating system uses the most up-to-date kernel without the need to reboot your server. After you purchase and install KernelCare, you can obtain and install the KernelCare "Extra" Patchset, which includes symlink protection.');
+        $promotion = $self->_lh->maketext('KernelCare provides an easy and effortless way to ensure that your operating system uses the most up-to-date kernel without the need to reboot your server.');
```

Comment:
Since this is pre-existing, we can leave this as is.  However, I would suggest using `[asis,KernelCare]`.